Misc/NEWS.d/next/Library/2020-08-14-14-31-04.bpo-39584.P1Rs2u.rst

```diff
@@ -0,0 +1,2 @@
+Prevent creating :class:`shared_memory.SharedMemory` objects with
+:code:`size` greater than 1TB.
```

Comment:
I don't think this is a valid use of the `code` role.

Reply:
Can you have a look at this: https://github.com/python/cpython/pull/21556#discussion_r461105702

Reply:
Hmm. Okay. But double backquotes are usually used in cases like this (as specified by the developer's guide https://devguide.python.org/documenting/#inline-markup).

Reply:
I guess both of them work. See this https://devguide.python.org/documenting/#module-specific-markup .
I was going for this because it is tried and tested.
Although, if you still want me to change to double back ticks, I will make changes.